docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,154 @@
+
+
+## Preparing to migrate
+
+The migration process requires changes in the channel configuration at /Channel/Orderer/ConsensusType and /Channel/Orderer/Orderers.
+In a JSON representation of the channel configuration, this would be .channel_group.groups.Orderer.values.ConsensusType and .channel_group.groups.Orderer.values.Orderers respectively.
+
+The `ConsensusType` is represented by three values: `Type`, `Metadata`, and `State`, where:
+- `Type` is either Raft (etcdraft) or BFT. This value can only be changed while in maintenance mode.
+- `Metadata` must carry valid Raft or BFT metadata.
+- `State` is the state of the system and can be one of the following: `STATE_NORMAL`, when the channel is processing transactions, or `STATE_MAINTENANCE` during the migration process.
+
+The `Orderers` is represented by one value, which is `ConsenterMapping`.
+Each consenter is represented by the following properties:
+* id
+* host
+* port
+* msp_id
+* identity
+* client_tls_cert
+* server_tls_cert
+
+\
+Before proceeding with the migration, the following preparatory steps are required:
+
+1.  **Config the number of node to be 3f + 1, so the cluster will be functional.**
+2. Prepare the BFT `Metadata` channel configuration.
+Refer to the [BFT configuration guide](https://hyperledger-fabric.readthedocs.io/en/latest/bft_configuration.html?highlight=BFT#configuring-and-operating-a-bft-ordering-service) for more information on the channel configuration fields.
+
+3. Prepare the ConsenterMapping. Ensure all nodes are part of the consenters. 
+
+Note: all the channels should receive the same BFT `Metadata` and `ConsentersMapping` configuration.
+
+4. Ensure all ordering service nodes are running the same version of Fabric, and that this version is v3.0.0 or greater.
+5. Ensure all peers are running at least v3.0.0 of Fabric. Make sure all channels are configured with the channel capability that enables migration.
+
+
+## Phase 1 - entry to maintenance mode
+Prior to setting the ordering service into maintenance mode, it is recommended that the peers and clients of the network be stopped.
+Leaving peers or clients up and running is safe, however, because the ordering service will reject all of their requests, their logs will fill with benign but misleading failures.
+
+The only field that has to be changed during this step is in the channel configuration at `/Channel/Orderer/ConsensusType`.
+In this step of the channel configuration update, only change the `State` from `STATE_NORMAL` to `STATE_MAINTENANCE`. 
+Do not change the `Type` or the `Metadata` field yet. Note that the Type should currently be Raft.
+
+After updating the `State`, verify that each ordering service node has entered maintenance mode on each of the channels. 
+This can be done by fetching the last config block and making sure that on each channel the `Type` is Raft, the `Metadata` matched the Raft metadata and the `State` is `STATE_MAINTENANCE`.
```

Comment:
```suggestion
This can be done by fetching the last config block and making sure that on each channel the `Type` is `etcdraft`, the `Metadata` matched the Raft metadata and the `State` is `STATE_MAINTENANCE`.
```